addon-test-support/extend/find-element.js

```diff
@@ -34,5 +35,10 @@ import { getExecutionContext } from '../-private/execution_context';
  * @throws Will throw an error if multiple elements are matched by selector and multiple option is not set
  */
 export function findElement(pageObjectNode, targetSelector, options = {}) {
+  deprecate('findElement is deprecated, please use findOne or findMany instead', false, {
+    id: 'ember-cli-page-object.old-finders',
+    until: '2.0.0'
+  });
```

Comment:
will need to add a deprecation section and `url` once https://github.com/san650/ember-cli-page-object/pull/465 is merged

Reply:
Merged! 

can you please add old finders to the new deprecation page and add an `url` into here?